totp/totp_test.go

```diff
@@ -135,4 +135,16 @@ func TestGenerate(t *testing.T) {
 	})
 	require.NoError(t, err, "generate larger TOTP")
 	require.Equal(t, 32, len(k.Secret()), "Secret is 32 bytes long as base32.")
+
+	k, err = Generate(GenerateOpts{
+		Issuer:      "SnakeOil",
+		AccountName: "alice@example.com",
+		SecretSize:  13, // anything that is not divisable by 5, really
+	})
+	require.NoError(t, err, "Secret size is valid when length not divisable by 5.")
+	require.NotContains(t, k.Secret(), "=", "Secret has no escaped characters.")
+}
+
+func TestSecretPadding(t *testing.T) {
+
```

Comment:
My bad. I started writing new test, but then figured that I can add new test case to existing test.

I have removed function and squashed commits to one to have clean history. I hope it is in ok now.